### PR TITLE
Add outdoor activity field to venue + refactor

### PR DESCRIPTION
**Pull request by Sartonon**
## Description

- Add outdoor activity checkbox to venue

- Refactor createOrUpdateVenue function (almost same function was in 2 places, now merged into one function)

### Closes
**[PT-511](https://helsinkisolutionoffice.atlassian.net/browse/PT-511):**

### CODE REVIEW

File: src/domain/event/EditEventPage.tsx [hunk 1]
```diff
@@ -135,6 +138,7 @@ const EditEventPage: React.FC = () => {
       goToEventDetailsPage();
     } catch (e) {
       if (process.env.NODE_ENV === 'test') {
```
Comment (jorilindell) [inline src/domain/event/EditEventPage.tsx:140]: Maybe helper function to test is it testing environment

File: src/domain/event/CreateEventPage.tsx [hunk 2]
```diff
@@ -119,6 +122,7 @@ const CreateEventPage: React.FC = () => {
     } catch (e) {
       // TODO: Improve error handling when API returns more informative errors
       if (process.env.NODE_ENV === 'test') {
```
Comment (jorilindell) [inline src/domain/event/CreateEventPage.tsx:124]: Maybe helper function to test is it testing environment

**Review by jorilindell — APPROVED** [review]
Looks good

File: src/domain/event/EditEventPage.tsx (hunk 1 above)
Comment (Sartonon) [reply src/domain/event/EditEventPage.tsx]: fixed

File: src/domain/event/CreateEventPage.tsx (hunk 2 above)
Comment (Sartonon) [reply src/domain/event/CreateEventPage.tsx]: fixed

**Pull request merged**